File: src/com/javaexamples/_10/example/CarpimTablosu.java
package com.javaexamples._10.example;

public class CarpimTablosu {

    // iç içe for döngüsü kullanarak çarpım tablosu oluşturma

    public static void main(String[] args) {

        for (int i = 1; i <= 10; i++) {
            for (int j = 1; j <= 10; j++) {
                System.out.println(i + " x " + j + " = " + (i * j));
            }
            System.out.println("**************");
        }
    }
}

File: src/com/kotlinexamples/_15/example/CarpimTablosu.kt
package com.kotlinexamples._15.example

// çarpım tablosunu oluşturunuz

fun main() {

    for (i in 1..10) {
        for (j in 1..10) {
            print("$i * $j = ${i * j}\t")
        }
        println()
        println()
    }
}